build_libs.py

```diff
@@ -16,7 +16,7 @@
 
 
 logger = logging.getLogger(__name__)
-cocotb_share_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), "share"))
+cocotb_share_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), "cocotb", "share"))
```

Comment:
We might want to consider moving this source code out of `cocotb` at some point too, or at least removing it from `package_data` in `setup.py` - today we install the source, which doesn't really make any sense now that installation builds binaries.

Reply:
I had thought about it too, but it's not pertinent for the changes I want to make immediately.

Reply:
Actually, it's pretty trivial, why not.

Reply:
Nvm, not trivial.